test/snoopr.jl

```diff
@@ -187,6 +188,14 @@ end
             Base.@irrational twoπ 6.2831853071795864769 2*big(π)
         end
     end
+
+    # Tabulate invalidations:
+    # Note: this table may change over time as invalidations get fixed.
+    SnoopCompile.report_invalidations(;
+        job_name = "job_name",
+        invalidations = invs,
+    )
+
```

Comment:
This only tests that it doesn't crash. Can we print to `io = IOBuffer()` and then test the output? It could be quite loose, e.g., `occursin("Invalidations %", str)` where `str = String(take!(io))`.

Reply:
Ah, yes, that'd be better.